github-for-jira.sd.yml

```diff
@@ -203,8 +203,55 @@ workers:
       instance: t2.small
       min: 1
       max: 5
-      #TODO Replace with queue size metrics when we switch to SQS
-      metrics: *CpuMemScalingRules
+      metrics: &CpuMemAndQueuesScalingRules
+        complexScalingRule:
+          EvaluationPeriods: 3
+          Threshold:
+            Lower: 30
+            Upper: 80
+          Metrics:
+            - Expression: "MAX([cpuUsage, memoryUsage, (pushQueueMessages/250)*100, (backfillQueueMessages/80)*100])"
```

Comment:
Here we calculate queues conditions along with memory and CPU. We evaluate every condition as a percent of how much it met. For CPU and memory it is a simply consumption. For queues is how much queue is filled up to the Threshold. 

For push queue we set threshold to 250, so when it is met on 80% (200 messages) we'll scale up.

For the backfill queue we use lower threshold because it tend to have less messages piling up (due to the difference in how this queue is used)